src/main/java/core/basesyntax/exeptions/FileDoesnExistsException.java

```
@@ -0,0 +1,6 @@
+package core.basesyntax.exeptions;
+public class FileDoesnExistsException extends RuntimeException {
+    public FileDoesnExistsException(String fileName) {
+        super(fileName);
```

This constructor forwards only the raw fileName as the exception message (super(fileName)). The checklist requires informative error messages when rethrowing exceptions: "Don't ignore exceptions in try-catch blocks. Don't leave e.printStackTrace() ... Let's rethrow a RuntimeException with an informative message and exception object." Change the message to something like: "File does not exist: " + fileName. Also add constructors that accept a cause so callers can preserve original exceptions (e.g., public FileDoesnExistsException(String message, Throwable cause) and public FileDoesnExistsException(Throwable cause)).